<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Clientform</class>
 <widget class="QDialog" name="Clientform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>995</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>1820</x>
      <y>0</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/logo.jpg);</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>161</width>
     <height>995</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));
border-top-right-radius: 50px;
color: white;
font: 75 20pt &quot;Berlin Sans FB&quot;;</string>
   </property>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);
border-radius: 10px;
color: white;</string>
    </property>
    <property name="text">
     <string>Auditeur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);
border-radius: 10px;
color: white;</string>
    </property>
    <property name="text">
     <string>Client</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);
border-radius: 10px;
color: white;</string>
    </property>
    <property name="text">
     <string>Rapport</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);
border-radius: 10px;
color: white;</string>
    </property>
    <property name="text">
     <string>Materielle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);
border-radius: 10px;
color: white;</string>
    </property>
    <property name="text">
     <string>Norme</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>89</y>
     <width>1759</width>
     <height>904</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>date_audit</string>
    </property>
   </widget>
   <widget class="QLabel" name="subject_lb">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>430</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>sujet : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_supprimer">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>140</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>supprimer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>adresse_email_client</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>date_email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="email_inp">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>380</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="email_lb">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Email : </string>
    </property>
   </widget>
   <widget class="QFrame" name="chart_frame">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>400</y>
      <width>541</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>num_client</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="lineEdit_dateAud">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>d/M/yyyy</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>adresse_client</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="lineEdit_dateEm">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>d/M/yyyy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_exporter">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>140</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>exporter</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="msg_inp">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>490</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>modifier</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>id_client</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>170</y>
      <width>541</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-color: rgb(255, 255, 0);
color: rgb(0, 0, 0);
selection-color: rgb(0, 0, 0);
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_adresseE">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="mail_error_lb">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>560</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_num">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ajouter">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>nom_client</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mail_send_btn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>540</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>envoyer</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nom">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_adresse">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Rechercher">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>chercher</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="subject_inp">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>430</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_date">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>270</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ID">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sort">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>afficher </string>
    </property>
   </widget>
   <widget class="QTableView" name="tab_cl">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>531</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="cal_tabView">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>430</y>
      <width>401</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="msg_lb">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>message : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tri">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>200</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Tri</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>210</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>par :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="client_tri_par_type_inp">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>210</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>id client</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>nom client</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>adresse client</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>num client</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>adresseemaill</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>date aud</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>date email</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
